index.html

```diff
@@ -61,16 +64,35 @@
     -->
     <link defer rel="import" href="../px-dark-theme/px-dark-theme-styles.html">
     <link defer rel="import" href="../px-dark-demo-theme/px-dark-demo-theme-styles.html">
-    <link defer rel="import" href="../px-demo/px-demo-theme-switcher.html">
+    <!-- <link defer rel="import" href="../px-demo/px-demo-theme-switcher.html">
```

Comment:
Is comment needed?

Reply:
Yeah, I left those intentionally. Probably some additional steps will be required here in order to integrate with https://predix-ui.com and this will be needed